GramAddict/core/storage.py

```diff
@@ -25,10 +22,8 @@ class Storage:
 
     def __init__(self, my_username):
         if my_username is None:
-            print(
-                COLOR_FAIL
-                + "No username, thus the script won't get access to interacted users and sessions data"
-                + COLOR_ENDC
+            logger.error(
+                "No username, thus the script won't get access to interacted users and sessions data"
```

Comment:
Unrelated to this commit, but I think we should raise an exception on this issue and not let the user continue. The reason being that it's not tracking any of the actions done and it leads to bad situations where users may have been followed by the script, but not tracked. Forcing a user to "unfollow any" in order to remove them - or do it manually. I can't think of any valid reasons why we would continue if we can't get their username.

If we do want to continue, we should at least safe it to an "undefined" user or something so that data isn't lost.

Reply:
Sure, can you open an issue so we can track this? Thanks.